admin/formatter/class-post-metabox-formatter.php

```diff
@@ -63,6 +65,22 @@ public function get_values() {
 		return $values;
 	}
 
+	/**
+	 * Gets the image URL for the post's social preview.
+	 *
+	 * @return string|null The image URL for the social preview.
+	 */
+	protected function get_image_url() {
+		$post_id = $this->post->ID;
+
+		if ( has_post_thumbnail( $post_id ) ) {
+			$featured_image_info = wp_get_attachment_image_src( get_post_thumbnail_id( $post_id ), 'single-post-thumbnail' );
```

Comment:
Not sure `single-post-thumbnail` is an existing attachment size. I guess it should be `thumbnail`? (which is the default). However: the `thumbnail` size is not guaranteed to be "square". Wondering if we should ask for a specific size e.g. `array( 150, 150 )`. Not a great expert of WP images, I'd recommend to discuss this internally.

Reply:
I've changed to `thumbnail`, since that indeed seems much more widely used. The Product team has indicated that for now it doesn't matter how the image is rendered exactly (i.e., cropped or squished), as long as it's consistent across posts and terms.